ci/validate_asciidoc.sh

```diff
@@ -57,6 +57,19 @@ do
     fi
   else
     #validate asciidoc
+
+    # Make sure include:: clauses are always more than one line away from the previous content
+    find "$dir" -name "*.adoc" -execdir sh -c 'grep -Pzl "\S\ninclude::" $1  | xargs -r -I@ realpath "$PWD/@"' shell {} \; > stuck
```

Comment:
A non-separated line with a trailing whitespace will escape this check
```
space is hereV
=== My header 
include::description.adoc[]
```
```suggestion
    find "$dir" -name "*.adoc" -execdir sh -c 'grep -Pzl "\S[ \t]*\ninclude::" $1  | xargs -r -I@ realpath "$PWD/@"' shell {} \; > stuck
```